andela/suyabay
# #121926873 Fix unaligned sign up button and remember me checkbox

#### What does this PR do?
- Fix the unaligned `remember me` checkbox in the signup and login pages
- Fix the improper positioning of the `sign up` and `sign in` buttons on the signup page and login page simultaneously.
#### Description of Task to be completed?
- Modify the `left` class by adding `remember-me` as part of the class.
- In the `main.css` file, style the remember me class by assigning it a margin-left of 7%.
- Modify the materialize `button` class by adding `signup-login-btn` as part of the materialize class button.
- In the `main.css` file, style the `signup-login-btn` by assigning it a margin-top of 3%.
#### How should this be manually tested?
- Go to `http://suyabay-staging.herokuapp.com/` then click on `login` link. This will take you to the login page. You should see the changes made on the `remember me` checkbox and the `sign in` button.
- Go to `http://suyabay-staging.herokuapp.com/` then click on `sign up` link. This will take you to the signup page. You should see the changes made on the `remember me` checkbox and the `sign up` button.
#### Any background context you want to provide?

none
#### What are the relevant pivotal tracker stories?
#121926873
#### Screenshots (if appropriate)

![screen shot 2016-06-21 at 11 42 12 am](https://cloud.githubusercontent.com/assets/14159420/16226856/47a49fdc-37a5-11e6-9424-c0bf99286266.png)

![screen shot 2016-06-21 at 11 42 27 am](https://cloud.githubusercontent.com/assets/14159420/16226857/47a4df56-37a5-11e6-8e5b-64ee1043be0a.png)
#### Questions:

none

[@unicodeveloper](https://github.com/unicodeveloper)
[@andela-oadebayo](https://github.com/andela-oadebayo)
